### PR TITLE
feat(groupinvite): use userpicker component

### PR DESCRIPTION
**Pull request by jasonwoodland**
**What this PR does** 📖
- Use the new user picker component in the group invite UI
- Add exclude prop to userpicker so we can hide members from the group invite UI

**Which issue(s) this PR fixes** 🔨
AP-1777

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: components/views/group/invite/Invite.vue [hunk 1]
```diff
@@ -1,31 +1,48 @@
...
 
-        await this.$store.dispatch('groups/sendGroupInvite', {
-          group: this.modal.group,
-          recipient: this.recipient,
-        })
+        await Promise.all(
+          this.recipients.map(
+            async (recipient) =>
+              await this.$store.dispatch('groups/sendGroupInvite', {
+                group: this.modal.group,
```
Comment (WanderingHogan) [inline components/views/group/invite/Invite.vue:41]: group does not exist

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
good when that one issue is accounted for

File: components/views/group/invite/Invite.vue (hunk 1 above)
Comment (jasonwoodland) [reply components/views/group/invite/Invite.vue]: Good spot

**Review by josephmcg — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

**Comment by stavares843** [comment]
Captura de ecrã 2022-06-15, às 20 31 31

if you click on add member without selecting any users, the picker just gets dismissed

File: components/views/navigation/toolbar/Toolbar.html [hunk 2]
```diff
@@ -100,7 +100,12 @@
     >
       <user-plus-icon size="1x" class="control-icon" />
     </div>
-    <GroupInvite v-click-outside="()=>toggleModal('groupInvite')" />
+    <GroupCreate v-click-outside="()=>toggleModal('creategroup')" />
+    <GroupInvite
+      v-if="modals.groupInvite.isOpen"
+      v-click-outside="()=>toggleModal('groupInvite')"
+      :group="recipient"
+    />
```
Comment (josephmcg) [inline components/views/navigation/toolbar/Toolbar.html:108]: I removed this GroupCreate component. it was the redundant one, we're replacing with quickChat at a later date

**Review by josephmcg — CHANGES_REQUESTED** [review]

**Review by iltumio — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**